app/scripts/modules/core/src/presentation/forms/inputs/NumberInput.tsx

```diff
@@ -24,11 +24,11 @@ export class NumberInput extends React.Component<INumberInputProps> {
   };
 
   public componentDidMount() {
-    this.props.validation.addValidator(this.validator);
+    this.props.validation && this.props.validation.addValidator(this.validator);
```

Comment:
@alanmquach I don't remember why I found I needed this but somewhere I was trying to use a `<NumberInput` and was hitting exceptions about `validation` not being present. Maybe I was doing something wrong?

Reply:
or @christopherthielen I dunno

Reply:
Are you using `NumberInput` somewhere other than a `FormikFormField`?

It looks like the `validation` props are provided by `FormikFormField` so when used like so
```js
<FormikFormField input={ props => <NumberInput {...props} /> } >
```
it's there.

With that said, since it's defined as optional in `IFormInputProps`, we probably should guard against it (or make it not optional).